test/output_stream_test.rb

```diff
@@ -6,6 +6,8 @@ class ZipOutputStreamTest < MiniTest::Test
   TEST_ZIP = TestZipFile::TEST_ZIP2.clone
   TEST_ZIP.zip_name = 'test/data/generated/output.zip'
 
+  ASCII8BIT = 'ASCII-8BIT'
```

Comment:
This constant can now be removed, but I'll tidy it up post-merge.

Reply:
Thx!